src/main/kotlin/com/atlassian/performance/tools/report/api/judge/MaximumCoverageJudge.kt

```diff
@@ -29,10 +29,12 @@ class MaximumCoverageJudge {
         criteria: PerformanceCriteria
     ): Verdict {
         val edibleBaseline = baseline.prepareForJudgement(
-            StandardTimeline(criteria.virtualUserLoad.total)
+            StandardTimeline(criteria.virtualUserLoad.total),
+            criteria.actionCriteria.mapValues { (_, actionCriteria) -> actionCriteria.outlierTrimming }
```

Comment:
You can compute trimmings once and reuse the same trimmings for baseline and experiment.